Northstar.Custom/mod/scripts/vscripts/sh_message_utils.gnut

```diff
@@ -168,6 +202,46 @@ int function NSGetPlayerResponse( entity player )
 	return server.playerPollResponses[ player ] - 1
 }
 
+string function NSCreateChoiceOnAllPlayers( string option1, string option2, float duration )
+{
+	string key = UniqueString()
+	foreach( player in GetPlayerArray() )
+	{
+		if ( !IsValid( player ) )
+			continue
+		NSCreateChoiceOnPlayer( player, option1, option2, duration, key )
+	}
+	return key
+}
+
+string function NSCreateChoiceOnPlayer( entity player, string option1, string option2, float duration, string key = "" )
+{
+	string id = key != "" ? key : UniqueString()
+
+	ServerToClientStringCommand( player, "ServerHUDMessagePut " + eDataType.CHOICE_OPTION + " " + option1 )
+	ServerToClientStringCommand( player, "ServerHUDMessagePut " + eDataType.CHOICE_OPTION + " " + option2 )
+	ServerToClientStringCommand( player, "ServerHUDMessagePut " + eDataType.CHOICE_DURATION + " " + duration )
+	ServerToClientStringCommand( player, "ServerHUDMessagePut " + eDataType.CHOICE_KEY + " " + id )
+
+	if( !( id in server.playerChoiceResponses ) )
+		server.playerChoiceResponses[id] <- {}
+	server.playerChoiceResponses[id][player] <- -1 // Reset choice response table entry
+	ServerToClientStringCommand( player, "ServerHUDMessageShow " + eMessageType.CHOICE )
+
+	return id
+}
+
+int function NSGetPlayerChoiceResponse( entity player, string key )
+{
+	if( !( key in server.playerChoiceResponses ) )
+		return -3
```

Comment:
Instead of integer return values we should go for named enums as they are more descriptive and make the code more readable.

Reply:
Adressed in 39cacda8f48ed963045f54c13146cba51e6cb851